src/Data/Functor/Product2.purs

```diff
@@ -0,0 +1,18 @@
+module Data.Functor.Product2 where
+
+import Prelude
+import Data.Functor.FunctorRight (class FunctorRight, rmap)
+
+-- | The Product of two `Bifunctor`s.
```

Comment:
Another reference to `` `Bifunctor` `` which should probably change to `two-parameter type constructors` or something?